setup.py

```diff
@@ -2,18 +2,15 @@
 from os import path
 from io import open
 
-import versioneer
-
 here = path.abspath(path.dirname(__file__))
 
 # Get the long description from the README file
 with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
 setup(name='pynvml',
```

Comment:
Given the simplicity of this code, think we can move this to a `pyproject.toml`, which will simplify and deduplicate dependency handling, metadata (like the README), etc.